zdg/mouse_up_event.py

```diff
@@ -0,0 +1,75 @@
+from zdg.event import Event
+from zdg.clicked_piece import ClickedPiece
+from zdg.move import Move
+import pdb
+
+
+class MouseUpEvent(Event):
+    """
+    Event: User Releases Mouse. If User was dragging 
+    a piece and its been moved to a valid location, we
+    can update the logic 
+
+    Args:
+        clicked_piece: previous cell that was clicked
```

Comment:
just curious, what was your rational of naming this clicked piece vs clicked cell?

Reply:
Ah I see, alright sounds good.